unit_test/Test_algorithms.py

```diff
@@ -6,6 +6,11 @@
 dirname = os.path.dirname(__file__)
 from train_simulation.Algorithms import Algorithms
 from train_simulation.Railway import Station, Connection, Stations, Connections, Lines
+from train_simulation.Moving import Moving, Train, Carrier
```

Comment:
This should be in another file. Create another file called Test_moving.py please

Reply:
You're right! Done